TemporaryParser/TPRBLiteralByteArrayErrorNode.class.st

```diff
@@ -0,0 +1,22 @@
+Class {
+	#name : #TPRBLiteralByteArrayErrorNode,
+	#superclass : #TPRBEnglobingErrorNode,
+	#category : #'TemporaryParser-Nodes'
+}
+
+{ #category : #'as yet unclassified' }
+TPRBLiteralByteArrayErrorNode class >> error: aToken withNodes: aCollection [
+	aCollection isEmpty
+		ifTrue: [ ^self new content: aCollection; start: aToken start; stop: aToken stop; errorMessage: ''']'' expected' ]
+		ifFalse: [ ^self new content: aCollection; start: aToken start; stop: aCollection last stop; errorMessage: ''']'' expected' ]
+]

```

Comment:
I see this same pattern in several englobing error nodes. Maybe this can be refactored?